js/util/lru_cache.js

```diff
@@ -82,7 +87,7 @@ class LRUCache {
      * @returns {Array<string>} an array of keys in this cache.
```

Comment:
In theory, if we upgrade to the latest `documentation` beta, we can strip out all return types for flow-annotated methods since they'll be inferred.